src/Network/WebSockets/Connection/Options.hs

```diff
@@ -31,6 +31,8 @@ data ConnectionOptions = ConnectionOptions
     { connectionOnPong                :: !(IO ())
       -- ^ Whenever a 'pong' is received, this IO action is executed. It can be
       -- used to tickle connections or fire missiles.
+    , connectionTimeout               :: !Int
+      -- ^ Timeout for connection establishment in microseconds.
```

Comment:
Seems like this comment is incorrect since we use `30 * 1000 * 1000` below?

Reply:
Not sure what you mean -  30 * 1000 * 1000 microseconds is 30s?

Reply:
Ah yes, I should have read this more carefully, apologies!